prover/scripts/integration_test

```diff
@@ -27,11 +28,50 @@ pushd _test_integration
     cid_evm=`upa compute circuit-id vk.json`
     cid_native=`prover universal-batch-verifier compute-circuit-id --app-vk=vk.json`
     if ! [ "${cid_evm}" == "${cid_native}" ] ; then
-        echo "cid_evm: ${cid_native}"
+        echo "cid_evm: ${cid_evm}"
         echo "cid_native: ${cid_native}"
         exit 1
     fi
 
+    # Check native and Typescript submission ids match
+    [ -e upa_config.json ] || cp ../tests/upa_config_2_with_submission_id.json upa_config.json
```

Comment:
Why do all this checking?  Did you mean to delete the entire dir above?  If so, the file will be newly copied.